packages/honeycomb-opentelemetry-web/src/console-trace-link-exporter.ts

```diff
@@ -23,13 +24,45 @@ export function configureConsoleTraceLinkExporter(
   options?: HoneycombOptions,
 ): SpanExporter {
   const apiKey = getTracesApiKey(options);
+  const { authRoot, uiRoot } = getUrlRoots(options?.endpoint);
   return new ConsoleTraceLinkExporter(
     options?.serviceName,
     apiKey,
     options?.logLevel,
+    authRoot,
+    uiRoot,
   );
 }
 
+export const getUrlRoots = (endpoint = '') => {
+  const url = new URL(endpoint);
+  const subdomainRegex = /(api)([.|-])?(.*?)(\.?)(honeycomb\.io)(.*)/;
```

Comment:
What are we trying to capture with the `(.*)` group at the end?

Reply:
I  don't think it's strictly needed, but it would match `v1/traces` at the end of the default `endpoint` value.